src/Directory.Build.targets

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project InitialTargets="AddSkipGetTargetFrameworkToProjectReferences">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), dir.targets))\dir.targets" />
+  <Target Name="_CheckForInvalidConfigurationAndPlatform" />
+  <Target Name="_CheckForUnsupportedTargetFramework" />
```

Comment:
Perhaps it is part of your buildtools changes but how are we blocking the import of the csharp targets?

Reply:
I haven't explicitly done this yet, maybe a change I will have to implement in buildtools.